### ISSUE TITLE
improving react-native-message-bar

### ISSUE THREAD

**Issue by Talor-A** [Talor-A/react-native-message-bar#1]
To anyone seeing this, I'm going to try to keep improving this library since the original author seems to have gone AWOL. (see [here](https://github.com/KBLNY/react-native-message-bar/issues/36))

- [ ] [adding close button](https://github.com/donovantc/react-native-message-bar/commit/ee4f315cec1dfcb65d21cc5cf3c7d2d402cfd522)
- [ ] [custom avatar](https://github.com/nikolay-radkov/react-native-message-bar/commit/61547ccf969d8df83c2e682a300b0e1cb07436f1)

**Comment by jeaye** [comment]
Can we get a separate NPM package for this? I can reference it from Github, but I'd prefer to stick with specific versions where I can.

**Comment by Talor-A** [comment]
Yeah I was thinking the same thing. I actually did a (very very simple) rewrite already that fixed a few major issues with the library, and I'd like to match the features this library has already. Got a suggestion for a title if I do create a new one? I also have never been a maintainer of an npm package before, but there's a first time for everything right?

**Comment by jeaye** [comment]
`react-native-alert-bar` comes to mind. Also `react-native-notification` and `react-native-notification-bar`.
